timeless/templates/views.py

```diff
@@ -1,2 +1,7 @@
+
 def order_by(query, params):
-    return query.order_by(*[param.replace(":", " ") for param in params])
\ No newline at end of file
+    return query.order_by(*[param.replace(":", " ") for param in params])
+
+
+def filter_by(query, params):
+    return query.filter_by(**(dict(p.split("=", maxsplit=1) for p in params)))
```

Comment:
We still need to implement generic way to filter queries. Now `filter_by` is implemented, but I guess, in reality, it will be never used. No one will be search table shapes by exact match of description. I suggest to start implementation some base `GenericFilter` class responsible for filtering. Ideally, by my opinion, for table shapes, it should be like:
```python
...
class TableShapeFilter(GenericFilter):
    id = filters.Equal()
    description = filters.Like()
```

So current implementation definitely works, but seems to me a bit useless.
@ekondrashev @emilianodellacasa what do you think?

Reply:
@vryazanov makes sense, I'll add a puzzle